tests/wpt/web-platform-tests/html/browsers/history/the-location-interface/location_origin.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<meta charset="utf-8">
+<title></title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  test(function () {
+    assert_equals(
+      location.origin,
+      location.protocol + '//' + location.host,
```

Comment:
Oh, good point. I ran `location.host` in the JS console in this window and saw no port number, so I assumed there would never be one.